### ISSUE TITLE
Are there methods to start and stop ssdp after creating

### ISSUE THREAD

**Issue by imnotbob** [achingbrain/ssdp#2]
My service stops advertising once it is connected via 
It was created via:
ssdpServer - ssdp(...)
ssdpeServer.advertise(...)

ssdpServer.stop()

Is there a method to continue/restart advertising?   would it be ssdpServer.start(), or some other cleanup / recreation?

**Comment by achingbrain** [comment]
Hmm, that's strange - it shouldn't stop advertising unless you explicitly tell it to.  Do you see any stack traces?

**Comment by imnotbob** [comment]
Sorry I was unclear.  The stop is what I coded, ie. it is a single use service at a time; when I connect I call stop 

My question is if I get other errors, is there a clean way (like ssdpServer.start() ?) to restart that service?   Currently I exit and let the service be restarted, but was hoping to lower the overhead of restarting the advertisement.

thx

**Comment by achingbrain** [comment]
TLDR: I would just call `bus.advertise` again.

There's not a lot of overhead in starting an advertisement - it's just a timer that periodically sends some UDP packets to the broadcast address.  It will listen on a socket to serve the device description document but you can get it not to do that by passing a `location` argument to `bus.advertise`.

https://github.com/achingbrain/ssdp#advertise-a-service
https://github.com/achingbrain/ssdp#full-api-and-options
